UnitTests/MonoDevelop.Debugger.Tests.TestApp/MonoDevelop.Debugger.Tests.TestApp.csproj

```diff
@@ -11,27 +11,6 @@
     <AssemblyName>MonoDevelop.Debugger.Tests.TestApp</AssemblyName>
     <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
   </PropertyGroup>
-  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|x86' ">
```

Comment:
Used by win32 md. Do not remove.

Reply:
fixed already, have been fighting git submodule :(